.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,42 @@
+name: Bug Report
+description: Report any errors or incorrect behaviors to help improve PyCytominer.
```

```suggestion
description: Report any errors or incorrect behaviors to help improve pycytominer.
```

pycytominer is lower case :)

---

Whoops! This always messes me up since the logo has capitals. I will fix this for all of templates.